README.md

```diff
@@ -33,7 +33,29 @@ And that's it. With this alone, you should be able to access Mission Control Job
 
 ### Authentication and base controller class
 
-By default, Mission Control's controllers will extend the host app's `ApplicationController`. If no authentication is enforced, `/jobs` will be available to everyone. You might want to implement some kind of authentication for this in your app. To make this easier, you can specify a different controller as the base class for Mission Control's controllers:
+By default, Mission Control's controllers will extend the host app's `ApplicationController`. If no authentication is enforced, `/jobs` will be available to everyone.
+
+#### HTTP authentication
+
+You can set a simple HTTP authentication by either using
+
+```ruby
+Rails.application.configure do
+  config.mission_control.jobs.http_auth_user = "captain"
+  config.mission_control.jobs.http_auth_password = "topsecret"
+end
+```
+or you can set it via ENV
+
+```shell
+ENV["MISSION_CONTROL_JOBS_HTTP_AUTH_USER"]=captain
+ENV["MISSION_CONTROL_JOBS_HTTP_AUTH_password"]=topsecret
+```
+If no value is provided (`nil`, `""` or `false`), authentication is skipped.
```

Comment:
Would you be up for changing this so that in production, if no configuration is set, it fails? The idea is that nobody could install Mission Control, forget to configure this, ship it to production and have this open to everyone by default.